.gitignore

```diff
@@ -12,3 +12,5 @@ yarn-error.log
 **/package-backup.json
 **/gen-webpack.config.js
 .DS_Store
+
+table-webview/out/
```

Comment:
just `out` would match better here imo. Why does this build to another folder anyways? I am comparing this to the other packages that can be used as dependencies in [klighd-vscode](https://github.com/kieler/klighd-vscode/tree/main/packages), where that is not necessary either. The executed scripts also seem to be a little different here.